go.mod

```diff
@@ -1,6 +1,6 @@
 module github.com/googlecloudplatform/gcsfuse
 
-go 1.21
+go 1.22.0
```

Comment:
This wouldn't work with `1.22` (ref [this](https://github.com/golang/go/issues/65568#issuecomment-1954876836) bug).

Other similar issues: https://github.com/golang/go/issues/62278, https://github.com/golang/go/issues/61455

Reply:
Could we please update the internal documentation to update this in case of minor version upgrade.

Reply:
@ashmeenkaur , Made the change you requested separately (outside the scope of this PR) as internal documentation is not in this repo.